package.json

```diff
@@ -2,9 +2,13 @@
   "name": "d3-metatable",
   "version": "0.4.0",
   "description": "a table element for heterogenous objects",
-  "main": "metatable.js",
+  "main": "build/d3-metatable.js",
+  "jsnext:main": "index",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "pretest": "rm -rf build && mkdir build && rollup -c",
+    "test": "tape 'test/**/*-test.js'",
+    "prepare": "npm run test && uglifyjs -b beautify=false build/d3-metatable.js -c -m -o build/d3-metatable.min.js",
+    "postpublish": "zip -j build/d3-metatable.zip -- LICENSE README.md build/d3-metatable.js build/d3-metatable.min.js"
```

Comment:
not sure why this generates a zip? GitHub releases will automatically provide zip files for tags, and I haven't seen zip files as being part of any module consumption.

Reply:
This was just copied from https://bost.ocks.org/mike/d3-plugin/. I don't see the necessity for it either.